base/libgit2/blob.jl

```diff
@@ -30,6 +36,14 @@ end
 
 Reads the file at `path` and adds it to the object database of `repo` as a loose blob.
 Returns the `GitHash` of the resulting blob.
+
+# Example
+
+```julia
+hash_string = hex(commit_oid)
+blob_file   = joinpath(repo_path,".git/objects", hash_string[1:2], hash_string[3:end])
```

Comment:
Extra spaces preceding the assignment intentional? Perhaps inject a space after the `repo_path` argument?